examples/LifeMonitorAPI.ipynb

```diff
@@ -0,0 +1,727 @@
+    }
+   ],
+   "source": [
+    "instance_uuid = instances['items'][0]['uuid']\n",
+    "response = s.get(f\"{lm_base_url}/instances/{instance_uuid}\")\n",
+    "assert response.status_code == 200, f\"Unexpected error {response.status_code}: {response.content}\"\n",
+    "instance = response.json()\n",
+    "instance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "List the latest builds for the chose instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'build_id': '769307075',\n",
+       "  'instance': {'managed': False,\n",
+       "   'name': 'test1_1',\n",
+       "   'resource': 'github/crs4/iwc',\n",
+       "   'roc_instance': '#test1_1',\n",
+       "   'service': {'type': 'travis',\n",
+       "    'url': 'https://travis-ci.org',\n",
+       "    'uuid': '0694c600-ffa9-4e37-bcd5-43a5790f415b'},\n",
+       "   'uuid': '045d99ee-1337-49bc-9465-31918bb655c3'},\n",
+       "  'status': 'passed',\n",
+       "  'suite_uuid': 'fbe0c0ae-c204-4543-ad30-b5b7d2efa875',\n",
+       "  'timestamp': '1620037573.0'},\n",
+       " {'build_id': '769294684',\n",
+       "  'instance': {'managed': False,\n",
+       "   'name': 'test1_1',\n",
+       "   'resource': 'github/crs4/iwc',\n",
+       "   'roc_instance': '#test1_1',\n",
+       "   'service': {'type': 'travis',\n",
+       "    'url': 'https://travis-ci.org',\n",
+       "    'uuid': '0694c600-ffa9-4e37-bcd5-43a5790f415b'},\n",
+       "   'uuid': '045d99ee-1337-49bc-9465-31918bb655c3'},\n",
+       "  'status': 'passed',\n",
+       "  'suite_uuid': 'fbe0c0ae-c204-4543-ad30-b5b7d2efa875',\n",
+       "  'timestamp': '1620027654.0'},\n",
+       " {'build_id': '763952272',\n",
+       "  'instance': {'managed': False,\n",
+       "   'name': 'test1_1',\n",
+       "   'resource': 'github/crs4/iwc',\n",
+       "   'roc_instance': '#test1_1',\n",
+       "   'service': {'type': 'travis',\n",
+       "    'url': 'https://travis-ci.org',\n",
+       "    'uuid': '0694c600-ffa9-4e37-bcd5-43a5790f415b'},\n",
+       "   'uuid': '045d99ee-1337-49bc-9465-31918bb655c3'},\n",
+       "  'status': 'passed',\n",
+       "  'suite_uuid': 'fbe0c0ae-c204-4543-ad30-b5b7d2efa875',\n",
+       "  'timestamp': '1616427012.0'}]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response = s.get(f\"{lm_base_url}/instances/{instance_uuid}/latest-builds\")\n",
+    "assert response.status_code == 200, f\"Unexpected error {response.status_code}: {response.content}\"\n",
+    "latest_builds = response.json()\n",
+    "for b in latest_builds['items']:\n",
+    "    del b['last_logs']\n",
+    "latest_builds['items'][:3]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pick a build a get the details."
```

Comment:
"a get" ---> "and get"

Reply:
Fixed in cb7090242725ef70b51c52e8c7decf80432836a6.